content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar aos serviços em execução no cluster Kubernetes.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Acessando serviços em execução no cluster
+
+No Kubernetes, [nós](/pt-br/docs/concepts/architecture/nodes/), [Pods](/docs/concepts/workloads/pods/) e [serviços](/docs/concepts/services-networking/service/) têm todos seus próprios IPs. Em muitos casos, os IPs dos nós, dos Pods e alguns dos IPs de serviço em um cluster não serão
```

Comment:
Sounds better to me "No Kubernetes, todos nós, Pods e serviços têm seus próprios IPs." instead of "No Kubernetes, nós, Pods e serviços todos têm seus próprios IPs.". Think about it.